app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ReviewsController < ApplicationController
+  before_filter :require_user
+  def create
+    @video = Video.find(params[:video_id])
+    review = @video.reviews.build(review_params.merge!(user: current_user))
+    if review.save
+      redirect_to @video
+    else
+      @reviews = @video.reviews.reload
+      render 'videos/show'
+    end
+  end
+
+  private
+
+  def review_params
+    params.require(:review).permit(:content, :rating, :video_id, :user_id)
```

Comment:
I am not sure about the `user_id` as permitted parameter. Are you assigning the user through form? if no then this is unnecessary leak. I see you merge in current user as user.